api/src/pcapi/settings.py

```diff
@@ -357,13 +358,15 @@
 
 # NOTION
 NOTION_TOKEN = secrets_utils.get("NOTION_TOKEN", "")
+NOTION_SYNCHRONIZATION_DISABLED = bool(os.environ.get("NOTION_SYNCHRONIZATION_DISABLED", False))
```

Comment:
(détails) Mon petit cerveau a du mal avec les double négations. ;) Et je propose un nom un peu plus explicite (quoique plus long) : 

```suggestion
SEND_SYNCHRONIZATION_ERRORS_TO_NOTION = bool(int(os.environ.get("SEND_SYNCHRONIZATION_ERRORS_TO_NOTION", 0)))
```

... et au passage, je vois que ce truc est activé pour les environnements hors-production, ce qui me semble une mauvaise idée, puisque c'est la même page Notion qui reçoit tout. On ne veut probablement pas mélanger les erreurs de staging (dans les rares cas où on y exécute une synchronisation) avec les erreurs de prod. En fait, on ne veut envoyer sur Notion que sur la prod, je pense. Je pourrai corriger dans un commit séparé.